entrypoint.sh

```diff
@@ -99,12 +109,14 @@ http {
 }
 EOF
 
-# Initial certificate request
-letsencrypt certonly \
-  --domain ${DOMAIN} \
-  --authenticator standalone \
-  ${SERVER} \
-  --email "${EMAIL}" --agree-tos
+# Initial certificate request, but skip if cached
+if [ ! -f /etc/letsencrypt/live/${DOMAIN}/fullchain.pem ]; then
+  letsencrypt certonly \
+    --domain ${DOMAIN} \
+    --authenticator standalone \
+    ${SERVER} \
+    --email "${EMAIL}" --agree-tos
+fi
```

Comment:
Since it's based on a specific base image, Alpine, it's safe to make shell assumptions (unlike the old days when I had to ensure Solaris compliance :( )